src/clj/puppetlabs/services/jruby/jruby_puppet_internal.clj

```diff
@@ -71,7 +100,7 @@
          (string? gem-home)]
    :post [(instance? ScriptingContainer %)]}
   (-> (ScriptingContainer. LocalContextScope/SINGLETHREAD)
-      (prep-scripting-container ruby-load-path gem-home)))
+    (prep-scripting-container ruby-load-path gem-home)))
```

Comment:
Oh, heh.  At least we don't have hash rockets to argue about, but I guess let bindings are effectively the same thing.